CoordImporter/Windows/MainWindow.cs

```diff
@@ -12,23 +12,29 @@
 using System.Collections.Immutable;
 using System.Linq;
 using System.Numerics;
+using Dalamud.Plugin.Services;
 
 namespace CoordImporter.Windows;
 
 public sealed class MainWindow : Window, IDisposable
 {
     private string textBuffer = string.Empty;
-    private HuntHelperManager huntHelperManager;
+    private IChatGui Chat;
+    private HuntHelperManager HuntHelperManager;
+    private IPluginLog Logger;
+    private Importer Importer;
 
-    public MainWindow(HuntHelperManager huntHelperManager) : base("Coordinate Importer")
+    public MainWindow(HuntHelperManager huntHelperManager, IChatGui chat, IPluginLog logger, Importer importer) : base("Coordinate Importer")
```

Comment:
similar nit to above: i vibe an order as probably logger, chat, importer, and then hunt helper manager.